codemods/bullmq/from-bull/src/types.ts

```diff
@@ -0,0 +1,7 @@
+import type core from 'jscodeshift';
+import type { Collection } from 'jscodeshift';
+
+export type ModifyFunction = (
```

Comment:
why is this export needed?
If you need it, you can make it a generic function with two arguments